src/init.cpp

```diff
@@ -1143,6 +1143,29 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                 return InitError(strprintf("Invalid network upgrade (%s)", vDeploymentParams[0]));
             }
         }
+
+        // To make testing easier (this code path is active only for regtest), activate missing network versions,
+        // so for example, if a Python (RPC) test does:
+        // extra_args = [
+        //    nuparams(BLOSSOM_BRANCH_ID, 205),
+        //    nuparams(HEARTWOOD_BRANCH_ID, 205),
+        //    nuparams(CANOPY_BRANCH_ID, 205),
+        //    nuparams(NU5_BRANCH_ID, 210),
```

Comment:
Add an RPC test that confirms setting `nuparams` for Heartwood and NU5 only, results in `getblockchaininfo` reporting that:
- Sprout, Overwinter, and Sapling are active from height 1 (due to test framework defaults).
- Blossom is active from the same height as Heartwood.
- Canopy is active from the same height as NU5.